### ISSUE TITLE
Extension does nothing

### ISSUE THREAD

**Issue by oshliaer** [funkyremi/vscode-google-translate#8]
It seems [google-translate-query] has an issue.

I've tried to debug

```js
function getTranslationPromise(selectedText, selectedLanguage, selection) {
  console.debug('getTranslationPromise', ...arguments);
  return new Promise((resolve, reject) => {
    translate(selectedText, { to: selectedLanguage })
      .then(res => {
        console.debug('getTranslationPromise => translate', ...arguments)
        if (!!res && !!res.text) {
          resolve(/** @type {TranslateRes} */{
            selection,
            translation: res.text
          });
        } else {
          reject(new Error("Google Translation API issue"));
        }
      })
      .catch(e => {
        console.error('getTranslationPromise => translate => error', ...arguments)
        reject(new Error("Google Translation API issue", e));
      });
  });
}
```

But sometimes I see only `console.debug('getTranslationPromise', ...arguments);` and `translate` doesn't resolve the Promise.

Any ideas how to debug this?

[google-translate-query]: https://github.com/seia-soto/google-translate-query

**Comment by funkyremi** [comment]
I've seen this too.
Can you try with other libraries ? 
https://www.npmjs.com/search?q=google%20translate

**Comment by oshliaer** [comment]
Yep, I thought about it. I'll try this week.

**Comment by funkyremi** [comment]
I published a new version 1.3.1 that seems to fix the issue. Please reopen if needed.

**Comment by oshliaer** [comment]
This is much better!
